packages/core-mobile/app/navigation/utils.ts

```diff
@@ -0,0 +1,17 @@
+import Logger from 'utils/Logger'
+import { navigate } from 'utils/Navigation'
+import AppNavigation from 'navigation/AppNavigation'
+import { NetworkTokensTabs } from 'screens/portfolio/network/NetworkTokens'
+
+export const navigateToChainPortfolio = async (): Promise<void> => {
+  //AppNavigation.Portfolio.NetworkTokens
```

Comment:
```suggestion
```

Reply:
removed in [Fix array and data type](https://github.com/ava-labs/core-mobile/pull/1659/commits/b1fb940934027accbc3947c307add309a5f5bf41)